Client/Experiments/Messaging/GleanPlumbMessageManager.swift

```diff
@@ -63,14 +63,15 @@ class GleanPlumbMessageManager: GleanPlumbMessageManagerProtocol {
     static let shared = GleanPlumbMessageManager()
 
     private let messagingUtility: GleanPlumbMessageUtility
-    private let messagingStore: GleanPlumbMessageStore
+    private let messagingStore: GleanPlumbMessagingStoreProtocol
```

Comment:
nit: I recommend we keep the naming consistent to `Message` instead of `Messaging` for the protocol.